src/gui_common/MainMenu.cs

```diff
@@ -313,4 +314,15 @@ private void OnReturnFromLoadGame()
 
         thriveLogo.Show();
     }
+
+    private void CallMiscTranslations()
+    {
+        _ = TranslationServer.Translate("OK");
+        _ = TranslationServer.Translate("Cancel");
+
+        foreach (var saveType in Enum.GetValues(typeof(SaveInformation.SaveType)).Cast<SaveInformation.SaveType>())
+        {
+            TranslationServer.Translate(SaveInformation.GetDescriptionForSaveType(saveType));
```

Comment:
I'm pretty sure this does *not* work correctly. The text extraction for `TranslationServer.Translate` is not executed. It's statically analysed by just grabbing a string inside that method call.

You need to manually list out all of the calls for all values for the strings used in SaveType. Might be worth moving the translate calls to that enum (if possible so that it is in the same place). If this is kept here then a big comment needs to be added to the enum to note that this needs to be updated.

Reply:
But it worked?! The enum values got correctly updated into the po files??

Reply:
@Kasterisk added `Description` to the list of extracted things. The extraction probably latched onto the "Description" attribute placed on the enum values, which I suppose is fine as well.

Reply:
So if I understand this correctly: Every enum value with a descriptionAttribute is automatically extracted and I dont need this foreach at all?

Reply:
I'm going to guess that's what happens, and you don't need this code.
babelrc file configures what extracted properties are looked at.